nautobot/extras/api/views.py

```diff
@@ -288,6 +289,11 @@ def run(self, request, class_path):
         if commit is None:
             commit = getattr(job_class.Meta, "commit_default", True)
 
+        try:
```

Comment:
Can/should we coerce non-dict `data` (empty string, or unspecified/`None`) to an empty dict here before calling `job.validate_data()`? I'm thinking about #732 here.